cedar-policy-validator/src/human_schema/to_json_schema.rs

```diff
@@ -604,11 +607,11 @@ fn make_warning_for_shadowing(n: &NamespaceRecord) -> impl Iterator<Item = Schem
     for (common_name, common_src_node) in n.common_types.iter() {
         // Check if it shadows a entity name in the same namespace
         if let Some(entity_src_node) = n.entities.get(common_name) {
-            let warning = SchemaWarning::ShadowsEntity {
+            let warning = SchemaWarning::ShadowsEntity(ShadowsEntityWarning {
```

Comment:
I've been adding `#[from]` onto the emum variants so that you can write `ShadowsEntityWarning { .. }.into()` without the outer wrapper.

Reply:
Done.